### ISSUE TITLE
Illegal callback invocation from native module. This callback type only permits a single invocation from native code.

### ISSUE THREAD

**Issue by adamivancza** [launchdarkly/react-native-client-sdk#190]
**Is this a support request?**
This issue tracker is maintained by LaunchDarkly SDK developers and is intended for feedback on the SDK code. If you're not sure whether the problem you are having is specifically related to the SDK, or to the LaunchDarkly service overall, it may be more appropriate to contact the LaunchDarkly support team; they can help to investigate the problem and will consult the SDK team if necessary. You can submit a support request by going [here](https://support.launchdarkly.com/hc/en-us/requests/new) or by emailing support@launchdarkly.com.

Note that issues filed on this issue tracker are publicly accessible. Do not provide any private account information on your issues. If your problem is specific to your account, you should submit a support request as described above.

**Describe the bug**
When evaluating a JSON flag there is an unhandled exception.
![Simulator Screen Shot - iPhone 11 - 2023-03-02 at 13 52 58](https://user-images.githubusercontent.com/19835917/222434135-e06ac109-f104-4f68-b223-9ee228058e4b.png)

**To reproduce**
Evaluate a JSON flag

**Expected behavior**
Do not crash when evaluating a JSON flag

**Logs**
-

**SDK version**
Started to happen in version 6.3.1, but I'm using 7.0.2 now

**Language version, developer tools**
For instance, Go 1.11 or Ruby 2.5.3. If you are using a language that requires a separate compiler, such as C, please include the name and version of the compiler too.

**OS/platform**
For instance, Ubuntu 16.04, Windows 10, or Android 4.0.3. If your code is running in a browser, please also include the browser type and version.

**Additional context**
Based on git blame this commit is causing this issue: https://github.com/launchdarkly/react-native-client-sdk/commit/a01ed3d49dd732311687732755700e18bdf7109c
I've compared how other flags are evaluated and there is a big difference in the JSON evaluation:
![SCR-20230302-jc7](https://user-images.githubusercontent.com/19835917/222434757-bf7bb846-4b60-48a5-a03e-981c25dba315.png)

Basically, I think we need to get rid of line 173 here: https://github.com/launchdarkly/react-native-client-sdk/commit/a01ed3d49dd732311687732755700e18bdf7109c#diff-cd97361c4f9361640dd6bed725225b77187ed86a1465713e326390be4ed932b6R173

**Comment by yusinto** [comment]
This has been fixed in [7.0.3](https://github.com/launchdarkly/react-native-client-sdk/releases/tag/7.0.3). Thanks @adamivancza for reporting and fixing this.
